### ISSUE TITLE
Accessing .toml keys from bridge/helpers/helper.go

### ISSUE THREAD

**Issue by zjoasan** [42wim/matterbridge#1289]
As a newbie to golang I have to little experience to figure out how to solve this. I've tried a few different approaches but none of them lets me read out a value from .toml. I've set a bool value in mattermost part of .toml "Slack2MMAttachments".

Most of the other code is done, link renameing the key title for the attachement from Slack titlelink to Mattermost title_link, and links in attachments are stored in Slack format and mattermost seems to be unabled to translate them.

If i can solve this last bit a mattermost-link to slack-link conversion in attachments should be as easy.

**Comment by qaisjp** [comment]
You probably want to pass in the boolean value into the helper! Do you have a branch or something I can look at?

**Comment by zjoasan** [comment]
Thank you for your reply., Well since I'm not that used to github, I haven't looked into branching and stuff, I've just committed changes from my local repository to my fork of matterbridge.

What I've done now to try to get it to work, is faking a a new config. and logger, to be able to access config.IsKeySet(). I just committed the last few hours work, to my fork.  Haven't even tested the code yet.

Work has been done on bridge/helper/helper.go, bridge/mattermost/mattermost.go, bridge/slack/slack.go bridge/config/config.go, matterclient/messages.go and matterbridge.toml.sample.

Still code all over the place since i''m still working on to get it as i want it, the link handling uis right now directly writen for a special github-bot on slacks messages that gets malformated on MM

[My fork](https://github.com/zjoasan/matterbridge)
